helm/CHANGELOG.md

```diff
@@ -0,0 +1,291 @@
+# Changelog
+
+This file documents all notable changes to [Gravitee.io Access Management 3.x](https://github.com/gravitee-io/helm-charts/tree/master/am/) Helm Chart. The release numbering uses [semantic versioning](http://semver.org).
+
+### 4.1.0
+
+To ensure the compatibility between the AM product and the AM Helm Chart, the versioning of the latter is changed.
+As of 4.1.0, the AM Helm Chart will follow AM release cycle.
+It means Helm Chart and AM should always be aligned.
+
+- [X] Add hook-delete-policy
+- [X] Allow users to use `logging.debug: true` when they define their own configuration file by defining a volume
+- [X] Remove ingress nginx annotation when `ingress.class` is not nginx
+
```

Comment:
Is it useful in the master branch as the same message is present for the 4.0.0 ?